ProcessMaker/processmaker
# FOUR-5996: Remove semantic versioning from upgrade migrator

## Issue & Reproduction Steps
Semantic versioning has been removed to simplify the operation of the upgrade migrations. This means the upgrade migrator will function in a similar way to the standard [Illuminate migration service provider](https://github.com/illuminate/database/blob/7eddad26821ed500c9725ba5655c095390fe2e5a/Migrations/Migrator.php).

## Solution
- Removed $to and $required properties on the stub and base upgrade migration file
- Refactored upgrade migration service provider, upgrade migrator, and related files to default to standard migration behavior
- Added `upgrade:reset` command to more closely resemble the standard migrations commands

## How to Test
- You should be able to run `php artisan upgrade` and see the upgrade migrations run in order in which they were created. 
- You should also be able to run `php artisan upgrade:reset` and see all of the previously run upgrade migrations being rolled back.

## Related Tickets & Packages
- JIRA ticket [FOUR-5996](https://processmaker.atlassian.net/browse/FOUR-5996)
- [connector-send-email PR](https://github.com/ProcessMaker/connector-send-email/pull/205)

## Code Review Checklist
